01 - Community/People/Gru80.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
This seems to be showing up files that I just reviewed in the previous PR...

For future PRs, this makes me think that it's easier to create a single PR with 2 separate commits - the all-automated bits, and then the manual bits...

Reply:
yeah, I forgot to say that this is based on the branch for #178. I just forgot to target the right branch.

Reply:
Thanks, I didn't know that was possible.